src/sage/libs/mpmath/ext_libmp.pyx

```diff
@@ -4,6 +4,7 @@ Faster versions of some key functions in mpmath.libmp
 from .ext_impl cimport *
 from sage.libs.gmp.all cimport *
 
+# the next line is used by mpmath
 from .ext_impl import exp_fixed, cos_sin_fixed, log_int_fixed
```

Comment:
I think some use the idiom `from .ext_impl import exp_fixed as exp_fixed, cos_sin_fixed as cos_sin_fixed, log_int_fixed as log_int_fixed` to explain that the purpose of the import is to fill the namespace

I don't know if this would suppress the cylint warning

Reply:
I do not think changing the import would be useful. Let just add a note to remember where this is used.